appserver/common/container-common/pom.xml

```diff
@@ -156,5 +156,10 @@
             <groupId>org.glassfish.soteria</groupId>
             <artifactId>jakarta.security.enterprise</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.concurro</groupId>
+            <artifactId>concurro</artifactId>
+            <version>3.1.0-SNAPSHOT</version>
```

Comment:
Please define using dependency management.
This presumably replaces the following
```
            <dependency>
                <groupId>org.glassfish</groupId>
                <artifactId>jakarta.enterprise.concurrent</artifactId>
                <version>${concurrent.version}</version>
            </dependency>
```

Reply:
Good point! It will require a bit more changes, as concurro is defined in BOM. Will fix.